From 46055872564f968dea7d03d79fc633b04e447c6e Mon Sep 17 00:00:00 2001
From: Juan Gutierrez <juan.gutierrez@nxp.com>
Date: Tue, 22 Nov 2016 15:07:33 -0600
Subject: [PATCH 1311/5242] MXSCM-225 dts: imx: fix the power gpio for usb
 otg2 on 6sxscm evb

commit  02b74a5cf19ed7c460d4c2976505068b7835470b from
https://source.codeaurora.org/external/imx/linux-imx.git

The usb otg2, on the SXSCM EVB board is powered up directly by the
GEN_V5 signal from the PMIC, so there is no gpio assigned.

The wrong assignation was preventing the busfreq driver to switch
to any other frequency, since the usb otg2 looks to be always
active.

Signed-off-by: Juan Gutierrez <juan.gutierrez@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sxscm-evb.dts |    1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6sxscm-evb.dts b/arch/arm/boot/dts/imx6sxscm-evb.dts
index 0b5052c..c1d0886 100644
--- a/arch/arm/boot/dts/imx6sxscm-evb.dts
+++ b/arch/arm/boot/dts/imx6sxscm-evb.dts
@@ -108,7 +108,6 @@
 			regulator-name = "usb_otg2_vbus";
 			regulator-min-microvolt = <5000000>;
 			regulator-max-microvolt = <5000000>;
-			gpio = <&gpio1 9 GPIO_ACTIVE_HIGH>;
 			enable-active-high;
 		};
 
-- 
1.7.9.5

